llvm/lib/LTO/ThinLTOCodeGenerator.cpp

```diff
@@ -1364,6 +1526,48 @@ ThinLTOCodeGenerator::writeGeneratedObject(StringRef OutputPath,
   return std::string(OutputPath);
 }
 
+template <class T>
```

Comment:
Nit: Move to a local header in `lib/LTO`

Reply:
It's only used in this .cpp file, so I'd prefer to keep it here to reduce the spread of our downstream diff.